openxc-ios-framework/VehicleManager.swift

```diff
@@ -12,6 +12,32 @@ import CoreBluetooth
 import ProtocolBuffers
 
 
+
+// FIXME: comparison operators with optionals were removed from the Swift Standard Libary.
+// Consider refactoring the code to use the non-optional operators.
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```

Comment:
what is this for?

Reply:
i have not added this.Ii am not aware of this method .I think this method is not used any where.